recipes/orfanage/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.4_9f713d2" %}
+{% set commit_id = "9f713d2a55065bb952c29eee1272ad9a9ca09ee2" %}
+
+package:
+  name: "orfanage"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/alevar/ORFanage.git
+  git_rev: {{ commit_id }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake >=3.2
+    - make
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - gsl
+    - libcblas
+    - zlib
+    - libbigwig
+
+test:
+  commands:
+      - orfanage --help > /dev/null
+
+about:
+  home: https://github.com/alevar/ORFanage
+  license: GPLv3
```

Comment:
can you please use a SPDX identifier for a license?

Reply:
Thank you. License tag is updated